import-scripts/oncokb-annotator.sh

```diff
@@ -78,6 +53,46 @@ FLOCK_FILEPATH="/data/portal-cron/cron-lock/oncokb-annotator.sh"
         return 1 # completion of import did not happen before timeout
     }
 
+    function gzip_oncokb_annotated_files() {
+        echo $(date)
+        echo "Gzipping data files for OncoKB Annotation for MSKSOLIDHEME"
+        if ! gzip -7 --force $ONCOKB_OUT_DIR/data*.txt ; then
+            message="Error during attempt to gzip $ONCOKB_OUT_DIR data files"
+            send_failure_messages "$message" "oncokb-annotator failed to run." "$message"
+            exit 1
+        fi
+    }
+
+    function rsync_latest_oncokb_annotated_msk_impact_to_clone() {
+        echo $(date)
+        echo "Rsyncing data files to clone of oncokb-annotated-msk-impact git repository"
+        rsync --exclude "/data/portal-cron/git-repos/oncokb-annotated-msk-impact/.git" -a $ONCOKB_OUT_DIR/ $ONCOKB_ANNOTATED_GIT_DIR
+    }
+
+    function commit_and_push_latest_oncokb_annotated_msk_impact() {
+        echo $(date)
+        echo "Committing (amending) and pushing data file changes to git repository"
+        wd=$(pwd)
+        cd $ONCOKB_ANNOTATED_GIT_DIR
+        # Create and push a git amended changeset
+        if ! git add . ; then
+            message="Error during attempt to 'git add .' in repository oncokb-annotated-msk-impact"
+            send_failure_messages "$message" "oncokb-annotator failed to run." "$message"
+            exit 1
+        fi
+        if ! git commit --amend --message "latest oncokb annotations of mskimpact" ; then
+            message="Error occurred during 'git commit --amend' ..."
+            send_failure_messages "$message" "oncokb-annotator failed to run." "$message"
+            exit 1
+        fi
+        if ! git push --force ; then
+            message="Error occurred during 'git push --force'"
+            send_failure_messages "$message" "oncokb-annotator failed to run." "$message"
+            exit 1
+        fi
+        cd $wd
+    fi
```

Comment:
looks like theres an extra closing if here, not sure if other code was dropped or missing. Causing unit test to fail.

Reply:
fixed